web/src/lib/api-client/apiClient.ts

```diff
@@ -37,6 +37,13 @@ export const getCompletedFiling = (): Promise<UserData> => {
   return get(`/completed-filing`);
 };
 
+export const postTaxRegistrationOnboarding = (props: {
+  businessName: string;
+  taxId: string;
+}): Promise<UserData> => {
+  return post(`/gov2go/onboarding`, props);
+};
```

Comment:
I'm noticing there is no apiClient function that calls the `/lookup` endpoint. Why does that endpoint need to exist?